src/aux-vm-browser/managers/SystemPortalCoordinator.ts

```diff
@@ -817,7 +817,7 @@ export class SystemPortalCoordinator<TSim extends BrowserSimulation>
                 ret.prefix = tagPrefix;
             }
 
-            if ((recentTagsCounts.get(`${tag}.${space}`) ?? 0) > 1) {
+            if (recentTagsCounts.get(`${tag}.${space}`)) {
```

Comment:
@TroyceGowdy Can you remove this if statement so that the code inside of it will always execute? Right now, if somehow `recentTagsCounts.get(`${tag}.${space}`)` ends up falsy, we'll end up returning `undefined` which might cause the system to error or crash.